examples/complex/esp32/ios-gateway/extra/iOS_IoTCore_Client_Demo/SwiftUIToUIKitViewController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  SwiftUIToUIKitViewController.swift
+//  UIKitSwiftUIDemo
+//
+//  Created by Danilo Campos on 6/16/20.
+//  Copyright © 2020 Danilo Campos. See license file.
```

Comment:
Just checking, what's the license?

Reply:
It didn't have a license but i didn't use any code from this file i deleted it before pushing it